src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -924,6 +931,25 @@ protected void disableOpenJ9SCC(List<String> dockerfileLines) {
         }
     }
 
+    protected void detectFeaturesSh(List<String> dockerfileLines) {
+        // Reset features.sh warning flag
+        shownFeaturesShWarning.set(false);
+
+        final String FEATURES_SH_COMMAND_LOWERCASE = "run features.sh";
```

Comment:
I dont think getCleanedLines() removes internal whitespace in the line so if the person types RUN<blank><blank>features.sh then the check should fail. Can we use a regex here?

Reply:
This is also not handled when detecting `RUN configure.sh`.  It may be better to change getCleanedLines() to remove internal whitespace. I'll open a separate issue.